recipes/ceci/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "ceci" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
```

```suggestion
  name: lsstdesc-{{ name|lower }}
```

We typically try to prefix packages on conda-forge to help with discoverability, but this is not a hard and fast rule ofc. What do you think?

---

Yep, makes sense. I don't anticipate much use of this outside DESC, though I guess you never know.